ocs_ci/deployment/deployment.py

```
@@ -4060,6 +4060,388 @@ def _patch_ramen_csv(self, csv_prefix, namespace, new_image):
+
+        logger.info("[ramenctl-e2e] Building config content with cluster details")
+        try:
+            ramenctl_config = self._build_ramenctl_config_content(ramenctl_dir)
+        except Exception as e:
+            logger.exception("[ramenctl-e2e] Failed to build ramenctl config")
+            return None
+
+        try:
+            with open(config_file, "w") as f:
+                yaml.dump(ramenctl_config, f, default_flow_style=False)
+            logger.info(f"[ramenctl-e2e] Updated config at {config_file}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)